### ISSUE TITLE
Design System [Component]: Multiple screens - Step 1, 6, 7, 8 - Radio buttons and checkboxes need fieldsets and legends

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#46128]
### Point of contact

Angela Fowler

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

On Step 1, 6, 7, 8, several of the radio button and checkbox components aren't contained in fieldsets.
For example, when navigating to the "relationship to deceased" radio button, the question isn't read. Similar issues occur on other steps as well.

### Reproduction steps

1. Go to one of the steps listed.
2. Inspect HTML.
3. Note that the radio button and checkbox question lack fieldsets.

### Proposed solution or next steps

Add the radio button and checkbox question to fieldsets. Like:

```html
<fieldset>
<lenged>Question text</legend>
…radio buttons…
</fieldset>
```

A bigger question? Is this not the default container for this component in the design system? Is that something we could make work out of the box when teams use radio button or checkbox components?

### References, articles, or WCAG support

WCAG 1.3.1 Info and Relationships

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by caw310** [comment]
This will carry over into Sprint 27 as engineers were working on other higher priority items.

**Comment by caw310** [comment]
Will carry over into the next sprint. A few developers were out of the office so this did not get worked on.

**Comment by Andrew565** [comment]
Looking at the comments here, as well as the current code, it looks like what should happen is for the va-radio and va-checkbox-group components to add `<fieldset>` to the V1-compatible versions, as we already have fieldset tags on the V3-compatible versions. @humancompanion-usds does this sound like a correct course of action to you?

**Comment by humancompanion-usds** [comment]
Yes, @Andrew565 that does sound correct to me. Glad to hear we already have these in v3 versions.

**Comment by caw310** [comment]
This will carry over due to an Axe check issue that is taking time to sort out.
